wpilibj/src/generated/main/java/edu/wpi/first/wpilibj/motorcontrol/SD540.java

```diff
@@ -18,19 +20,19 @@
  * Mindsensors.
  *
  * <ul>
- *   <li>2.05ms = full "forward"
- *   <li>1.55ms = the "high end" of the deadband range
- *   <li>1.50ms = center of the deadband range (off)
- *   <li>1.44ms = the "low end" of the deadband range
- *   <li>0.94ms = full "reverse"
+ *   <li>2.050ms = full "forward"
+ *   <li>1.550ms = the "high end" of the deadband range
+ *   <li>1.500ms = center of the deadband range (off)
+ *   <li>1.440ms = the "low end" of the deadband range
+ *   <li>0.940ms = full "reverse"
```

Comment:
(Same as C++)
Is this precise to the thousandths place? (It's not the biggest deal, especially since changing this would make the jinja more complicated)